src/HIE/Bios.hs

```diff
@@ -1,12 +1,11 @@
 -- | The HIE Bios
 
 module HIE.Bios (
-  -- * Find and load a Cradle
+  -- * Find and load a Cradle and its configuration
     Cradle(..)
-  , findCradle
   , loadCradle
-  , loadImplicitCradle
-  , defaultCradle
```

Comment:
Why is this deleted?

Reply:
Their only use was from `Main.hs` and I thought it would be better to encapsulate Cradle selection to a single function. Do you want me to revert it?